src/System.Runtime.TimeZoneData/System.Runtime.TestData.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
the name of this file is wrong, it should be `System.Runtime.TimeZoneData.csproj`